app/components/autocomplete/emoji_suggestion/emoji_suggestion.tsx

```diff
@@ -2,8 +2,10 @@
 // See LICENSE.txt for license information.
 
 import Fuse from 'fuse.js';
-import React, {useCallback, useEffect, useMemo, useRef} from 'react';
+import {debounce} from 'lodash';
```

Comment:
We have a `debounce` in our codebase (`app/helpers/api/general.ts`). Not sure what are the main differences between both, but should we try to use only one of them? Or there are reasons to still have them both?

Reply:
yes we do have it in our `helpers` and I do believe we should stick with the one from `lodash` as it has the cancel and flush methods. The helper function has a way to invoke immediately but that is the same as `flush`